app/views/shared/_head.html.erb

```diff
@@ -6,4 +6,5 @@
   <%= csp_meta_tag %>
   <%= stylesheet_link_tag "application" %>
   <%= javascript_include_tag "application" %>
+  <script defer data-domain="data-catalogue-prototypes-main.herokuapp.com/manage-prototype" src="https://plausible.io/js/script.js"></script>
```

Comment:
The domain here is referring to Lewis' prototypes, should it be instead the domain for whichever environment it's in? Presumably we'll only want Plausible running in the environment we use for production?